Source/Scene/Globe.js

```diff
@@ -465,6 +465,30 @@ define([
             tileProvider.oceanNormalMap = this._oceanNormalMap;
             tileProvider.enableLighting = this.enableLighting;
 
+            surface.initialize(frameState);
+        }
+    };
+
+    /**
+     * @private
+     */
+    Globe.prototype.update = function(frameState) {
+        if (!this.show) {
+            return;
+        }
+
+        var context = frameState.context;
+        var width = context.drawingBufferWidth;
+        var height = context.drawingBufferHeight;
+
+        if (width === 0 || height === 0) {
+            return;
+        }
```

Comment:
Also perhaps this should be a helper function since it comes up 3 times.

```javascript
        if (!this.show) {
            return;
        }

        var context = frameState.context;
        var width = context.drawingBufferWidth;
        var height = context.drawingBufferHeight;

        if (width === 0 || height === 0) {
            return;
        }
```